client/components/accountoverview/accountOverviewCardV2.tsx

```diff
@@ -185,8 +187,105 @@ export const AccountOverviewCardV2 = ({
 		}
 	`;
 
+	const benefitsCss = css`
+		${textSans.medium()};
+		list-style: none;
+		margin: ${space[5]}px 0 ${space[4]}px -4px;
+		padding: 0;
+
+		li + li {
+			margin-top: ${space[2]}px;
+		}
+
+		li {
+			display: flex;
+			align-items: flex-start;
+		}
+
+		svg {
+			flex-shrink: 0;
+			margin-right: ${space[2]}px;
+			fill: ${palette.brand[500]};
+		}
+	`;
+
+	const benefitsButtonCss = css`
+		${textSans.small()}
+		margin-top: ${space[1]}px;
+		padding: 0;
+		color: ${palette.brand[500]};
+		border-bottom: 1px solid ${palette.brand[500]};
+	`;
+
 	return (
 		<Card heading={productTitle}>
+			{specificProductType === PRODUCT_TYPES.supporterplus &&
+				nextPaymentDetails && (
+					<Card.Section backgroundColor="#edf5fA">
+						<p
+							css={css`
+								${textSans.medium()}
+								margin: 0;
+								max-width: 35ch;
+							`}
+						>
+							You’re supporting the Guardian with a{' '}
+							{nextPaymentDetails.paymentValue} per{' '}
+							{nextPaymentDetails.paymentInterval} support and
+							extra benefits.
+						</p>
+						<ul
+							id="benefits"
+							css={benefitsCss}
+							hidden={!showBenefits}
+						>
+							<li>
+								<SvgTickRound size="small" />
+								<span>
+									<strong>
+										A regular supporter newsletter.
+									</strong>{' '}
+									Get exclusive insight from our newsroom
+								</span>
+							</li>
+							<li>
+								<SvgTickRound size="small" />
+								<span>
+									<strong>Uninterrupted reading.</strong> See
```

Comment:
Some of these are using {' '} to create a space and some are just on the same line. Should we keep it consistent?